packages/manager-react-components/src/hooks/services/mocks/services.mock.ts

```diff
@@ -12,11 +16,99 @@ export const servicesMockErrors = {
   getDetails: 'Get services details error',
 };
 
+export const defaultServiceResponse: ServiceDetails = {
```

Comment:
It'll be better to separate `services.handler.ts` and `services.mock.ts`

Reply:
Done but I had to modify the files in other apps